src/notation/qml/MuseScore/NotationScene/NotationToolBar.qml

```diff
@@ -85,12 +85,15 @@ Item {
             navigation.panel: root.navigationPanel
             navigation.name: toolTipTitle
             navigation.order: model.index
+            accessible.name: (item.checkable ? (isMixerToggled ? item.title + qsTrc("  ", " On ", "  ") :
```

Comment:
You can use `item.checked` here too - `toggle-mixer` action is checkable

Reply:
item.checked is not working here because mixer button is not remain in checked state when button is clicked , it is always remain in non checked state. That's why I use bool here.

Reply:
This state must be there
You can verify this by turning the mixer on/off from the toolbar and comparing the state of the Mixer item from the View menu (from app menu)
We need to investigate why it doesn’t update its state for toolbar

Reply:
@Eism I've verified that when I click the "Mixer" button in the Notation Toolbar, the state reflects in the View menu. However, the state is not updating in the Notation Toolbar for the Mixer button accessibility. I tried implementing `item.checked` and also the `isChecked` property, which is implemented in StyledMenuItem.qml for view menu, but this is not working. Can you suggest the right way to fix this?

Reply:
I found the problem

NotationToolBarModel is inherited from AbstractMenuModel, like AppMenuModel. All the logic for updating the state of items is located inside AbstractMenuModel, and all the children should properly configure this model and use it. The problem is that NotationToolBarModel was configured incorrectly - it has an overridden load method but didn't call the load method of the parent, where the state (checked) of items is updated. 

To solve this issue, you need to call AbstractMenuModel::load at the end of NotationToolBarModel::load, like in AppMenuModel